Assets/tiny_rogue/Scripts/Components/Mobile.cs

```diff
@@ -7,6 +7,7 @@ public struct Mobile : IComponentData
     {
         public float3 Initial;
         public float3 Destination;
+        public WorldCoord DestWc;
```

Comment:
I'm not sure how nested IComponentData is handled by ecs. Maybe replace with an int2?